schemas/context/profile.schema.json

```diff
@@ -80,17 +80,17 @@
             "$ref": "https://ns.adobe.com/xdm/context/pushnotificationtoken"
           }
         },
-        "xdm:orgUnitID": {
```

Comment:
This is a breaking change, please update the `CHANGELOG.md`

Reply:
OK, breaking changes between releases. Got it.

Reply:
Please see fa2c6f4 for how to document the breaking part of the change.

Reply:
thanks. keeping in mind for future.